### DISCUSSION TITLE
Plugin for right-click on item in headline list?

### DISCUSSION THREAD

**Discussion by xavierISd** [lwindolf/liferea#1004] [Q&A]
Is it possible for a plugin to add an item to the context menu that appears when right-clicking on an item's headline? It'd be on the menu that starts with "Open in Tab" and ends with "Remove Item" (see screenshot, below). I believe the menu is generated by [ui_popup_item_menu](https://github.com/lwindolf/liferea/blob/71beee94a212387917a47e21f338b20e35e7c3ce/src/ui/popup_menu.c#L86).

I'm new to GTK introspection, but I'm also conceptually not sure how to find a context menu that's context-sensitive and doesn't have a permanent "home" I can look at with GTK Inspector.

![context-menu](https://user-images.githubusercontent.com/60920846/121429442-c8870280-c92b-11eb-9f03-ecf81d0bbfb4.png)

**Comment by mozbugbox** [comment]
It's not possible.  

See:

https://github.com/lwindolf/liferea/blob/c6189118b75d99c83ef38de93efc6b91a4e62bf5/src/webkit/liferea_web_view.c#L243-L250
